src/settings_internal.js

```diff
@@ -226,4 +226,9 @@ var TRANSPILE_TO_ES5 = false;
 
 // A copy of the default the default INCOMING_MODULE_JS_API. (Soon to
 // include additional items).
-var ALL_INCOMING_MODULE_JS_API = []
+var ALL_INCOMING_MODULE_JS_API = [];
+
+// List of all imports that are weak, and therefore allowed to be undefined
```

Comment:
Maybe it's worth mentioning this only applies to dynamic libraries (since weak imports aren't a thing for executables, right?)

Reply:
Done.  Although this is an internal-only setting so this is not user-facing documentation.